plugins/modules/remote_management/redfish/ilo_redfish_info.py

```diff
@@ -4,9 +4,10 @@
 # GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
 # SPDX-License-Identifier: GPL-3.0-or-later
 from __future__ import absolute_import, division, print_function
+
 __metaclass__ = type
 
-DOCUMENTATION = '''
+DOCUMENTATION = """
```

Comment:
Generally such reformatting changes should be avoided, especially in an already this large PR. It makes it a lot harder to review this, and thus will increase the time it takes to get this merged.

Reply:
Hi, since the module_utils, i.e. ilo_redfish_utils.py is common to all three modules, should it be included in all three PRs?